diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -416,6 +417,8 @@ namespace diff_drive_controller
     controller_nh.param("angular/z/max_jerk"               , limiter_ang_.max_jerk               ,  limiter_ang_.max_jerk              );
     controller_nh.param("angular/z/min_jerk"               , limiter_ang_.min_jerk               , -limiter_ang_.max_jerk              );
 
+    controller_nh.param("limit_accel_before_wheel_speed_limiter", limit_accel_before_wheel_speed_limiter_, false);
```

Comment:
may be best to use the already initialized `limit_accel_before_wheel_speed_limiter_` as the default value here.